ChangeLog.d/fix-ilp32.txt

```diff
@@ -0,0 +1,4 @@
+Bugfix
+   * Fix a compile failure in the constant_time module when building
+     for watchos (i.e. for Aarch64 ILP32). Reported by Paulo Coutinho
```

Comment:
Minor, perhaps updating just for that: watchos is not really material here, what's material is arm64_32. Suggested wording: “when building for arm64_32, e.g. for watchOS”.